client/src/App.js

```diff
@@ -4,13 +4,15 @@ import Layout from "./components/layout";
 
 import About from "./pages/About";
 import Home from "./pages/Home";
+import DisplayData from "./pages/DisplayData";
 
 const App = () => (
 	<Chakra>
 		<Layout>
 			<Routes>
 				<Route path="/" element={<Home />} />
 				<Route path="/about/this/site" element={<About />} />
+				<Route path="/display" element={<DisplayData />} />
```

Comment:
Can we rename the component do "Schools" or "DisplaySchools", we will be displaying different data all over the app so we need to give meaningful names :) Route can be called "/schools"

Reply:
Done